managed/src/main/java/com/yugabyte/yw/controllers/EncryptionAtRestController.java

```diff
@@ -179,7 +180,7 @@ public Result deleteKMSConfig(UUID customerUUID, UUID configUUID) {
         } catch (Exception e) {
             final String errMsg = "Error caught attempting to delete KMS configuration";
             LOG.error(errMsg, e);
-            return ApiResponse.error(BAD_REQUEST, e.getMessage());
+          throw new YWServiceException(BAD_REQUEST, e.getMessage());
```

Comment:
Same suggestions as other diffs.
why is this BAD_REQUEST?
Is there exception thrown deep in the code that shoul instead be changed to as YWE?

Reply:
Not really. We can remove this try catch block from everywhere. I was just worried about logging error message but its fine as discussed. Will make changes accordingly. Thanks!

Reply:
Done. Thanks!